tests/smoke.rs

```diff
@@ -445,6 +458,32 @@ fn dependencies() {
     run!()
 }
 
+/// Test building a dependency specified as `dependencies` in Xargo.toml
+#[cfg(feature = "dev")]
+#[test]
+fn dependencies_from_working_dir() {
+    fn run() -> Result<()> {
+        // need this exact target name to get the right gcc flags
+        const TARGET: &'static str = "thumbv6m-none-eabi";
```

Comment:
Each test needs to have a different `TARGET`, to avoid using the same output directories. But the good part is that you can replace `none` by whatever you want and the targets still work -- see what the other tests are doing.

Reply:
Ack.